.travis-deploy.sh

```diff
@@ -44,7 +44,7 @@ fi
 
 # Add new content
 mv "${srcpath}"/build/* .
-cp "${srcpath}"/deploy-settings.json settings.json
+cp "${srcpath}"/.deploy-settings.toml settings.toml
```

Comment:
Why starting with a dot? Not that I mind, I just don't see why.

Reply:
Because it's irrelevant for usually development so having it hidden means that people are not wondering what this is about.
The file should be pretty irrelevant for almost all people after all